2/TEXT/PL/ARIPCL01.TXT

```diff
@@ -0,0 +1,11 @@
+[0:00:000] Na wieczność zostanę związana, porzuciwszy nadzieję, a mój duch przykuty do tego miejsca...
+[0:07:000]
+[0:08:734] Bowiem, dopiero co umarłszy, niewrażliwa już byłam na krzywdę.
```

Comment:
Zmiana wprowadzona nie w tej linii tekstu, co trzeba :P
Tu powinno zostać "To, co zniszczyło Krąg, nie mogło mnie tknąć."